src/eureka/S5_lightcurve_fitting/models/SinusoidPhaseCurve.py

```diff
@@ -188,6 +182,6 @@ def eval(self, channel=None, **kwargs):
         if self.eclipse_model is None:
             eclipse = 1
```

Comment:
Presumably, if there is no eclipse model then there probably is no `fp`.  In which case, the PC amplitude is now in units of Fp/Fs (i.e., hundreds of ppm) instead of being a fraction of Fp (e.g., ~0.5).  I think this is ok as is.

Reply:
Sure, that can work too